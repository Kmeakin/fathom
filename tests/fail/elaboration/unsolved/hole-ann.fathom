//~ allow-errors = true

?woopsie : Type
