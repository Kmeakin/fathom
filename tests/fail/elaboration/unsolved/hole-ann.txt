//~ exit-code = 1

?woopsie : Type
