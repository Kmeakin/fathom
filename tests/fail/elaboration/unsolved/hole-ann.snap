stdout = '''
?woopsie : Type
'''
stderr = '''
error: failed to infer hole expression
  ┌─ tests/fail/elaboration/unsolved/hole-ann.fathom:3:1
  │
3 │ ?woopsie : Type
  │ ^^^^^^^^ unsolved hole expression

'''
